### PR TITLE
Added more error checking and tests for draw card

### PR DESCRIPTION
**Pull request by mdangnguyen**
Added jest as devDependencies

[DD-13](https://github.com/suryarawat/DigitalDeck/issues/14)

### CODE REVIEW

File: digital-deck-server/__tests__/playerTests.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ beforeAll(() => {
```
Comment (dhillonJas) [inline digital-deck-server/__tests__/playerTests.js:29]: can you add the play card test here as well?

Comment (mdangnguyen) [reply digital-deck-server/__tests__/playerTests.js:29]: Sure. You want me to just grab them from your old PR?

Comment (dhillonJas) [reply digital-deck-server/__tests__/playerTests.js:29]: yeah, that would be great. You might have to change initial value of playerCard and tableCard though

Comment (dhillonJas) [reply digital-deck-server/__tests__/playerTests.js:29]: Never mind, I'll add them. I need to add some error checking when handling request. I'll create a PR for that

**Review by dhillonJas — APPROVED** [review]

File: digital-deck-server/__tests__/playerTests.js (hunk 1 above)
Comment (mdangnguyen) [reply digital-deck-server/__tests__/playerTests.js:29]: Sounds good

**Review by hasan-umanitoba — APPROVED** [review]

**Pull request merged**